extensions/amp-story/1.0/animation.js

```diff
@@ -117,6 +119,10 @@ class AnimationRunner {
     /** @private @const */
     this.duration_ = animationDef.duration || this.presetDef_.duration || 0;
 
+    /** @private @const */
+    this.easing_ = animationDef.easing || this.presetDef_.easing ||
+        'cubic-bezier(0.4, 0.0, 0.2, 1)';
```

Comment:
nit: move the `cubic-bezier(0.4, 0.0, 0.2, 1)` to a constant (`DEFAULT_EASING`?)